config/routes.rb

```diff
@@ -5,7 +5,10 @@
   # Defines the root path route ("/")
   # root "articles#index"
   root to: 'users#index'
+  resources :comments, only: [:new, :create]
+  resources :likes, only: [:create]
+
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
-    end
+    resources :posts, only: [:index, :show, :new, :create]
+  end
```

Comment:
- [ ] Please note that the comments and like should be nested in the post path so that you can get the correct route to the comments and likes according to the project. The comments routes should be `/posts/:post_id/comments` and the likes routes should be `/posts/:post_id/likes`. Consider the routes below;
```ruby
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show, :new, :create] do
      resources :comments, only: [:new, :create]
      resources :likes, only: [:create]
    end
  end
```


This is your current routes
![image](https://user-images.githubusercontent.com/94551484/230681315-8c67c414-6d5a-4c61-a5b0-f730741e733d.png)